src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -808,6 +808,261 @@ public void isSentToMultiSig_whenNonStandardErpRedeemScript_shouldReturnTrue() {
         assertScriptIsMultiSig(nonStandardErpRedeemScript);
     }
 
+    @Test
+    public void getSigsPrefixCount_whenP2PKScript_shouldReturnZero() {
+        BtcECKey signer = new BtcECKey();
```

Comment:
Preferably, let's use fixed key values so the tests are deterministic

Reply:
done